.github/workflows/package-Extensions.PersistentStorage.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       run: dotnet test test/${{env.PROJECT}}.Tests
 
     - name: dotnet pack ${{env.PROJECT}}
-      run: dotnet pack src/${{env.PROJECT}} --configuration Release --no-build
+      run: dotnet pack src/${{env.PROJECT}} --configuration Release #--no-build <- OpenTelemetry.Extensions has a conditional net6.0 target which causes dotnet pack to break when -no-build is used (for some reason)
```

Comment:
The conditional target framework is not added for this project though.

Reply:
added